system-tests/e2e-suite/src/test/scala/com/hotels/styx/proxy/HttpOutboundMessageLoggingSpec.scala

```diff
@@ -112,10 +112,10 @@ class HttpOutboundMessageLoggingSpec extends FunSpec
         assertThat(logger.log.size(), is(2))
 
         assertThat(logger.log(), hasItem(loggingEvent(INFO,
-          "requestId=[-a-z0-9]+, request=\\{method=GET, uri=http://localhost:[0-9]+/foobar, origin=\"localhost:[0-9]+\", headers=\\[.*\\]}")))
+          "requestId=[-a-z0-9]+, request=LiveHttpRequest\\{version=HTTP/1.1, method=GET, uri=http://localhost:[0-9]+/foobar, headers=\\[.*\\], id=[-a-z0-9]+}, origin=appOne:generic-app-01:localhost:[0-9]+")))
```

Comment:
I see we are now exposing class names (implementation detail) in our access logs (a documented feature). 

Let's remove them from the access logs. The class names are okay for error logs etc but not here. As a documented feature we need to consider the access logs like any other external interface with possible backwards compatibility implications. 

Therefore please remove the class name from this message.

Also, I'd move the `origin` either in its original place, or in front of the request message. My reasoning is:

1) Trying to keep the message format as similar as possible 

2) Headers field is often really large, and therefore should appear after all "fixed size" attributes. This is because it is difficult to find and see the fixed size attributes after a large body of header text.

Reply:
The origin isn't part of the request, so I moved it out.  Good point about long headers field.  I'll move it to in front.

Reply:
> The origin isn't part of the request, so I moved it out.

Yeah, it was a right thing to do.

Reply:
moved to front